src/Controllers/HealthzController.cs

```diff
@@ -59,7 +59,7 @@ public async Task<IActionResult> RunHealthzAsync()
                 StatusCode = res.Status == HealthStatus.Unhealthy ? (int)System.Net.HttpStatusCode.ServiceUnavailable : (int)System.Net.HttpStatusCode.OK,
             };
 
-            Response.Headers.Add(CpuCounter.CapacityHeader, $"current={CpuCounter.CpuPercent}, target={App.Config.BurstTarget}, max={App.Config.BurstMax}");
+            Response.Headers.Add(CpuCounter.CapacityHeader, $"service={App.Config.BurstService}, current-load={CpuCounter.CpuPercent}, target-load={App.Config.BurstTarget}, max-load={App.Config.BurstMax}");
```

Comment:
The header is output everytime. Should I add a config to enable/disable the header?

Reply:
Open an issue in triage for it.  I think we should.

Reply:
it's in the healthz controller, so it's only output on /healthz/* calls.  I think that's what we want.